screen-onboarding/src/main/java/com/ivy/onboarding/OnboardingEvent.kt

```diff
@@ -0,0 +1,26 @@
+package com.ivy.onboarding
+
+import com.ivy.legacy.datamodel.Account
+import com.ivy.legacy.datamodel.Category
+import com.ivy.wallet.domain.data.IvyCurrency
+import com.ivy.wallet.domain.deprecated.logic.model.CreateAccountData
+import com.ivy.wallet.domain.deprecated.logic.model.CreateCategoryData
+
+sealed interface OnboardingEvent {
+
+    data object loginWithGoogle : OnboardingEvent
```

Comment:
Events are classes and must start with upper-case. like `LoginWithGoogle`. Also the login with Google feature is gone and you can delete that code